execute/tokendata/usdc/http.go

```diff
@@ -196,3 +202,5 @@ func (h *httpClient) inCoolDownPeriod() bool {
 	defer h.coolDownMu.RUnlock()
 	return time.Now().Before(h.coolDownUntil)
 }
+
+var _ HTTPClient = (*httpClient)(nil)
```

Comment:
compliance check not required, constructor returns an interface instance, it should fail there